common-app-lib/src/cmr/common_app/services/search/elastic_search_index.clj

```diff
@@ -109,7 +172,7 @@
 (defmethod send-query-to-elastic :unlimited
   [context query]
   (when (:aggregations query)
-    (e/internal-error! "Aggregations are not supported with queries with an unlimited page size."))
+    (errors/internal-error! "Aggregations are not supported with queries with an unlimited page size."))
```

Comment:
Thanks for using a more readable ns var :-) (i.e., +1 to ` [cmr.common.services.errors :as errors]`)